Directory.Packages.props

```diff
@@ -15,13 +15,14 @@
     <PackageVersion Include="Microsoft.SqlServer.Server" Version="1.0.0" />
     <PackageVersion Include="Microsoft.SqlServer.TransactSql.ScriptDom" Version="$(ScriptDomPackageVersion)" />
     <PackageVersion Include="Microsoft.SqlServer.Types" Version="160.1000.6" />
+    <PackageVersion Include="Nuget.Protocol" Version="6.13.2" />
```

Comment:
this is a third-party library. it might change the license for the SDK

Reply:
@llali  NuGet.Packaging was already included.

Reply:
We are taking on an additional runtime dependency to Nuget API, but because all we're shipping is a MSBuild task, the Nuget assemblies should already be there when the task runs. This is why we don't need to package any additional assemblies into the SDK.

Reply:
even if you don't include it in the package I think we need to include it in the license. @dzsquared am I wrong here?